sig-tencentcloud/README.md

```diff
@@ -0,0 +1,27 @@
+# SIG: TencentCloud
+
+This SIG focuses on the current TencentCloud provider for Spinnaker.
+
+## Meetings
+
+* Regular SIG Meeting
+  * Last Thursday of every month, at 11 AM Asia/Shanghai ([See in your time zone](https://www.thetimezoneconverter.com/?t=11am&tz=Shanghai))
+  
+  [Agenda](https://docs.google.com/document/d/1BKBzlLh6k5TVi-fEjKy0be3TQtduGgiA03_-g9O7jwM/edit?usp=sharing)
+  [Agenda (qq-docs)](https://docs.qq.com/doc/DZnVDVW1XbURtcnhm)
```

Comment:
Probably just having one agenda doc is enough?

Reply:
I assumed this is because Google Docs aren't available everywhere?

Reply:
But it seems like qq.com is (I can see the linked doc, anyway) so probably just having a single doc is better?